src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -282,6 +282,35 @@ private void testEditQuestionAction() throws Exception {
                                 .contains("The recipient's team members can see your response, but not the name of the recipient, or your name."));
         
         feedbackEditPage.clickAndCancel(feedbackEditPage.getCancelQuestionLink(-1));
+        
+        ______TS("add question 2 and edit it to giver's team members and giver");
+        feedbackEditPage.clickAddQuestionButton();
+        feedbackEditPage.fillQuestionBox("test visibility when choosing giver's team members and giver");
+        feedbackEditPage.selectGiverToBeStudents();
+        feedbackEditPage.selectRecipientsToBeGiverTeamMembersAndGiver();
+        feedbackEditPage.clickMaxNumberOfRecipientsButton();
+        feedbackEditPage.clickAddQuestionButton();
+        
+        ______TS("test Recipient's Team Members row is hidden");
+        feedbackEditPage.clickVisibilityOptionsForQuestion(2);
+        
+        assertEquals("Recipient's Team Members", feedbackEditPage.getVisibilityOptionTableRow(2, 4).getAttribute("textContent").trim());
```

Comment:
@Sumei1009, Let's add a comment, or extract this out to another function, to explain why this needs to be done this way instead of the usual getText